cloud_init.tf

```diff
@@ -7,7 +7,23 @@ data "template_cloudinit_config" "cloud_init_tasks" {
   part {
     content_type = "text/x-shellscript"
     content = templatefile(
-      "${path.module}/scripts/setup_freeipa.sh", {
+      "${path.module}/cloud-init/download-certificates.py", {
+        cert_bucket_name   = "cool-certificates"
```

Comment:
Is there a reason that you don't have `cool-certificates` as a terraform variable (e.g. `var.cert_bucket_name`)?  That's how we do it in [openvpn-server-tf-module](https://github.com/cisagov/openvpn-server-tf-module/blob/develop/cloud_init.tf#L22).

Reply:
Remedied in commit b8b030d.

This was something I intended to go back and fix but forgot about.  I also created a variable for the PKCS#12 certificate password in commit 6fc041b.